cypress/e2e/swap.test.ts

```diff
@@ -431,16 +312,14 @@ describe('Swap', () => {
           // Open the currency select modal.
           cy.get('#swap-currency-output .open-currency-select-button').click()
 
-          // Wait for the currency list to load
-          cy.contains('1inch').should('exist')
-
           // Select UNI as output token
           cy.get(getTestSelector('token-search-input')).clear().type('Uniswap')
           cy.get(getTestSelector('currency-list-wrapper'))
             .contains(/^Uniswap$/)
             .first()
-            .should('exist')
```

Comment:
this `should` was here to allow time for the list to load--are you sure it's not necessary?

Reply:
I removed this and saw no difference in the test.